lib/perl/Genome/Model/Tools/Vcf/EvaluateVcf.pm

```diff
@@ -472,8 +472,11 @@ sub stat_sensitivity_exact {
     my $results = $self->get_rawstats();
 
     my $total_true = $self->stat_total_true_positive_exact;
-    if ($total_true == 0) {
-        return 0;
+    {
+        no warnings;
```

Comment:
After thinking about this and conversing on the side via IM, I think I'll keep these "no warnings" sections as is for the moment.